src/components/MainPageCaroussel.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import { Carousel, IconButton } from '@material-tailwind/react';
+import { BiRightArrow, BiLeftArrow } from 'react-icons/bi';
+import { v4 as uuidv4 } from 'uuid';
+import DoctorCard from './DoctorCard';
+
+const MainPageCaroussel = ({
+  isLoading,
+  error,
+  doctorsGroup,
+}) => (
+  <Carousel
```

Comment:
There are overlapping cards when moving the carousel

![Screenshot 2023-09-30 091830](https://github.com/abel-tefera/doctor-appointment-front-end/assets/40035531/0facfe13-a676-456a-b479-9f7fc9c94a85)